server/messages.go

```diff
@@ -141,20 +145,24 @@ func processMessages(conn net.Conn, c chan *packetInfo, t *Tracker) {
 		}
 
 		if err := scanner.Err(); err != nil {
-			fmt.Fprintf(os.Stderr, "[Error] %s\n", err.Error())
-			break
+			fmt.Fprintf(os.Stderr, "[Error] Message scanner: %s\n", err)
+			return
 		}
 
 		if mb.Len() == 0 {
-			break
+			fmt.Fprint(os.Stderr, "[Error] 0-length message")
```

Comment:
This needs a trailing newline. `Fprint` doesn't do that for you like you think it would. You can even see it in the travis test output.

Reply:
Thanks for catching those.